pages/agent/v3/securing.md.erb

```diff
@@ -100,6 +100,51 @@ if [[ "${BUILDKITE_PLUGINS:-}" != "" ]]; then
   exit 1
 fi
 ```
+  
+### Whitelisting Plugins
+
+Per above, whitelisting can be done with the [agent’s environment hook](hooks). It may be desirable to whitelist a set of plugins that agents are allowed to execute.
```

Comment:
Thanks for providing some updated code and putting the time to put together a PR, I helped write the [initial version](https://gist.github.com/toolmantim/e52f86da6525829cd49557d813830386) of this ☺️🙌🏼

The reason I didn't add it to the docs right away, was that it was untested, and a little too complicated to use as a copy+paste example. And it would at least need some tests.

I've just written tests and published the initial version here (with some fixes):
https://github.com/buildkite/buildkite-plugin-whitelister

Did you want to collaborate on that repo, to integrate your changes? And then we can link to that from the doc, rather than inlining the code?

Reply:
Yeah awesome, much better idea. Can also start to do complex things like version pinning that way 😁

Reply:
Happy if I update the docs to reference ☝️? Or would you prefer I close this PR? (I'm happy either way)

Reply:
Yeah, let's update the docs to reference that repo I reckon!